doc/source/serve/autoscaling-guide.md

```diff
@@ -0,0 +1,369 @@
+
+![users](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_users.png)
+
+The results of the load test are as follows:
+
+::::{tab-set}
+:::{tab-item} Replicas
+<img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_replicas.png" alt="replicas" width="400"/>
+:::
+
+:::{tab-item} QPS
+<img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_rps.svg" alt="qps" width="400"/>
+:::
+
+:::{tab-item} Latency
+<img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_latency.svg" alt="latency" width="400"/>
+:::
+::::
+
+The results of the load test are as follows:
+
+|  |  |  |
+| -------- | --- | ------- |
+| Replicas | <img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_replicas.png" alt="replicas" width="450"/> |
+| QPS | <img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_rps.svg" alt="qps" width="450"/> |
+| P50 Latency | <img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_latency.svg" alt="latency" width="450"/> |
+
+
+<!-- | ![replicas](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_replicas.png){width=200px} | ![qps](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_rps.png){width=200px} | ![latency](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_latency.svg){width=200px} | -->
+
+Notice that the number of autoscaled replicas closely matches the number of Locust users over time. This makes sense because you can simplify it to one user is mapped to one deployment replica, and the user will continuously send the right amount of traffic to keep one replica busy. As expected, the throughput of the system also increases with the number of users and replicas. Lastly, the latency sees some spikes when the traffic increases, but otherwise stays relatively steady. Notice that the biggest latency spike happens at the beginning of the test, because the deployment starts with 0 replicas.
+
+
+## Model Composition Example
+
+It is less straightforward to determine an autoscaling configuration for a multi-model application because different deployments will autoscale differently. Every workload will have a different latency and differing levels of concurrency. As a result, it takes experimentation to find the right autoscaling config for a model-composition application.
+
+We will go through an example of a simple application with three deployments composed together to build some intuition about multi-model autoscaling. Suppose we have deployments:
+* `HeavyLoad`: A mock 200ms workload with high CPU usage.
```

Comment:
[Nit] Can we rename:
* `HeavyLoad` -> `Slow`
* `LightLoad` -> `Fast`

That way we can say "the `Slow` deployment" or "the `Fast` deployment" whenever we talk about them.

Open to keeping it the same or renaming to something else though.

Reply:
That's a good idea, except the images need the deployment name labels for clarity, so I would have to rerun the tests to get the new images 😅 for the sake of getting this cherry picked ASAP, leaning towards keeping the deployment names this time!

Reply:
Good call, let's keep it the same.